i18n/ind/docusaurus-plugin-content-docs/current/docs-user/staking/early-unlock.mdx

```diff
@@ -1,57 +1,55 @@
+    'buka cepat', 
+    'membuka lebih awal', 
+    'buka darurat', 
+    'modifikasi staking', 
+    'modifikasi posisi kunci', 
+    'update posisi kunci', 
+  ] 
+--- 
 
-import positions from '../../../../../../static/img/docs/staking/positions.png';
-import fastUnlock from '../../../../../../static/img/docs/staking/fast-unlock.png';
-import fastUnlocked from '../../../../../../static/img/docs/staking/fast-unlocked.png';
-import fastReview from '../../../../../../static/img/docs/staking/fast-review.png';
+import positions from '../../../../../../static/img/docs/staking/positions.png'; 
+import fastUnlock from '../../../../../../static/img/docs/staking/fast-unlock.png'; 
+import fastUnlocked from '../../../../../../static/img/docs/staking/fast-unlocked.png'; 
+import fastReview from '../../../../../../static/img/docs/staking/fast-review.png'; 
 
-This guide explains step-by-step how to unstake a [locking position](overview#locking-positions) before it has expired.
+Panduan ini menjelaskan langkah demi langkah cara membuka staking pada [posisi kunci](overview#locking-positions) sebelum waktu kedaluwarsanya.
 
-:::warning
-Early unstaking involves a **penalty fee**.
+:::warning 
+Membuka staking lebih awal melibatkan **biaya penalti**.
 
-The penalty fee is calculated based on the remaining duration of the locking position and the amount of LSK being unstaked.
-The penalty fee is deducted from the LSK being unstaked.
+Biaya penalti dihitung berdasarkan sisa durasi posisi kunci dan jumlah LSK yang akan dibuka stakingnya. 
+Biaya penalti akan dipotong dari LSK yang dibuka.
 
-If you simply want to unstake normally, please refer to the guide on [How to unstake](stake-unstake#how-to-unstake-your-lsk-tokens).
+Jika Anda hanya ingin membuka staking secara normal, harap merujuk ke panduan [Cara Membuka Staking](stake-unstake#how-to-unstake-your-lsk-tokens).
 
-For more details check [Early Unlock](overview#early-unlock).
+Untuk informasi lebih lanjut, lihat [Early Unlock](overview#early-unlock).
 :::
 
-1. Go to your [Staking Dashboard](https://portal.lisk.com/staking/dashboard) in the Lisk Portal.
-    Connect your wallet if you haven't already.
-2. In the list of your locking positions, click on the locking position you wish to unstake early, to expand it.
+1. Masuk ke [Dashboard Staking](https://portal.lisk.com/staking/dashboard) di Portal Lisk. 
+   Hubungkan dompet Anda jika belum dilakukan.
+2. Di daftar posisi kunci Anda, klik posisi kunci yang ingin Anda buka stakingnya lebih awal untuk memperluasnya. 
     <img src={positions} /><br />
-    :::tip
-    You can do early unstake for paused and unpaused locking positions alike.
+    :::tip 
+    Anda dapat membuka staking lebih awal untuk posisi kunci yang dijeda maupun yang tidak dijeda. 
     :::
-3. Click on the `Early Unlock` button.
-4. Confirm you understand the consequences of early unstaking.
-    Review the amount of LSK you will receive after the penalty fee is deducted.
```

Comment:
@TalhaMaliktz The translation can be corrected into: "Review the amount of LSK you will receive after deducting the penalty fee."